src/main/kotlin/racingCar/Main.kt

```diff
@@ -0,0 +1,9 @@
+package racingCar
+
+import racingCar.lib.UserInputData
+import racingCar.ui.InputView
+
+fun main() {
+    val userInput: UserInputData = UserInputData(InputView().getNumOfCarsInput(), InputView().getNumOfMovesInput())
```

Comment:
```suggestion
    val userInput = UserInputData(
        InputView().getNumOfCarsInput(),
        InputView().getNumOfMovesInput()
    )
```
가로로 길어질 경우에는, 개행을 해주면 가독성이 좋아져요